notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -126,6 +136,21 @@ define([
             'q' : 'jupyter-notebook:close-pager',
         };
     };
+    
+    KeyboardManager.prototype.get_default_vim_command_shortcuts = function() {
+      var shortcut = this.get_default_command_shortcuts();
+      delete shortcut['down'];
+      shortcut['ctrl-down'] = 'jupyter-notebook:select-next-cell';
+      delete shortcut['up'];
+      shortcut['ctrl-up'] = 'jupyter-notebook:select-previous-cell';
```

Comment:
I dont' think we want to bind to `ctrl-something`, it will  screw up with windows manager on some linux/osx